domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -311,5 +313,10 @@ public SampleClaim withDirectionsQuestionnaireDeadline(LocalDate dqDeadline) {
         this.directionsQuestionnaireDeadline = dqDeadline;
         return this;
     }
+
+    public SampleClaim withMoneyReceivedOn(LocalDate moneyReceivedOn) {
```

Comment:
With Lombok annotation work which @geek-soft did lately, we no longer require withXXX method to populate variables. I think you'll once you merge with master as PR you can refactor it then.